gbfs.md

```diff
@@ -273,9 +273,17 @@ Field Name | Required | Type | Defines
 `vehicle_types` | Yes | Array | Array that contains one object per vehicle type in the system as defined below.
 \- `vehicle_type_id` | Yes | ID | Unique identifier of a vehicle type. See [Field Types](#field-types) above for ID field requirements.
 \- `form_factor` | Yes | Enum | The vehicle's general form factor. <br /><br />Current valid values are:<br /><ul><li>`bicycle`</li><li>`car`</li><li>`moped`</li><li>`other`</li><li>`scooter`</li></ul>
-\- `propulsion_type` | Yes | Enum | The primary propulsion type of the vehicle. <br /><br />Current valid values are:<br /><ul><li>`human` _(Pedal or foot propulsion)_</li><li>`electric_assist` _(Provides power only alongside human propulsion)_</li><li>`electric` _(Contains throttle mode with a battery-powered motor)_</li><li>`combustion` _(Contains throttle mode with a gas engine-powered motor)_</li></ul> This field was inspired by, but differs from the propulsion types field described in the [Open Mobility Foundation Mobility Data Specification](https://github.com/openmobilityfoundation/mobility-data-specification/blob/master/provider/README.md#propulsion-types).
+\- `car_type` | Optional | Enum | The type of car. <br /><br />Current valid issues are: <br /><ul><li>`microcar` _(smallest size of car, engine smaller than 700cc, e.g Smart Fortwo)_ </li><li>`subcompact` _(sedan with interior volume index no larger than 99 cu ft, e.g. Kia Rio)_</li><li>`compact` _(sedan with interior volume index no larger than 109 cu ft, e.g. Hyundai Elantra)_ </li><li>`mid-size` _(sedan with interior volume index no larger than 119 u ft, e.g. Toyota Camry)_</li><li>`full_size` _(sedan with interior volume index larger than 120 cu ft, e.g. Tesla Model S)_</li><li>`station-wagon` _(large cargo area with rear hinged tailgate, e.g. Subaru Outback)_ </li><li>`suv` _(similar to station-wagon with a light truck frame, e.g. Honda Pilot)_ </li><li>`minivan` _(also known as multi-purpose vehicle or people mover, e.g. Dodge Grand Caravan)_ </li><li>`pickup truck` _(enclosed cab with open cargo area and tailgate, e.g. Chevrolet Ram)_</li><li>`other`
```

Comment:
Can this be merged with `form_factor`? I feel setting `form_factor: "car"` and `car_type: "full_size"` a bit duplicated. The fields could be renamed to `car_*` and then be used like this: `form_factor: "car_full_size"` or `form_factor: "car_suv"`.